.github/workflows/CI.yml

```diff
@@ -17,21 +17,91 @@ on:
   # Also build when some pull requests are created
   pull_request:
 
-  # Allows you to run this workflow manually from the Actions tab
+  # Allows you to run this workflow manually from the Actions tab by
+  # selecting CI and then "Run workflow" menu on the right branch
+  # and clicking on "launch_tmate_terminal_for_debug".
+  # Unfortunately this works only for the default branch.
+  # So you can either
+  # - change the default branch of the PR on the GitHub repository owning the PR
+  #   and launching in Actions tab;
+  # - or edit directly the step below which runs tmate and push to the
+  #   PR, ignoring the manual workflow launch.
   workflow_dispatch:
+    inputs:
+      launch_tmate_terminal_for_debug:
+        type: boolean
+        description: 'Run the build with tmate debugging enabled (https://github.com/marketplace/actions/debugging-with-tmate)'
+        required: false
+        default: false
 
 jobs:
   build:
     # The type of runner that the job will run on
     runs-on: ubuntu-latest
 
-    # Use the Khronos container with the asciidoctor toolchain preinstalled.  We
-    # reference the image by its SHA rather than its tag because they sometimes
-    # overwrite a tag with a different image (which has a different SHA).  This
-    # SHA corresponds to tag "asciidoctor-spec.20240727".
-    container: khronosgroup/docker-images@sha256:089687083ceb36483a3917389e4278718ab19c594099634f5dd80e22540c960f
+    # Use Khronos container with asciidoctor toolchain preinstalled
+    container: khronosgroup/docker-images:asciidoctor-spec
```

Comment:
This looks like a merge mistake.

Reply:
I removed this from my PR as I thought it was too complex.